internal/server/data/deviceflowauthrequest.go

```diff
@@ -92,24 +88,29 @@ func GetDeviceFlowAuthRequest(tx ReadTxn, opts GetDeviceFlowAuthRequestOptions)
 	return (*models.DeviceFlowAuthRequest)(rec), nil
 }
 
-func SetDeviceFlowAuthRequestAccessKey(tx WriteTxn, dfarID uid.ID, accessKey *models.AccessKey) error {
-	_, err := tx.Exec(`
-		UPDATE device_flow_auth_requests
-		SET 
-			access_key_id = ?,
-			access_key_token = ?
-		WHERE id = ?
-	`, accessKey.ID, models.EncryptedAtRest(accessKey.Token()), dfarID)
+func DeleteExpiredDeviceFlowAuthRequests(tx WriteTxn) error {
+	query := querybuilder.New("UPDATE device_flow_auth_requests")
```

Comment:
Shouldn't this be `DELETE FROM device_flow_auth_requests`...? I thought we wanted to hard delete here so that the table doesn't keep filling up (i.e. soft delete in `DeleteDeviceFlowAuthRequest()` and then reap in this function).

Reply:
Yes good catch! Thanks. This must have been from in a bad rebase